Gemfile.lock

```diff
@@ -75,6 +80,10 @@ GEM
     erubis (2.7.0)
     fakeredis (0.5.0)
       redis (~> 3.0)
+    faraday (0.9.2)
+      multipart-post (>= 1.2, < 3)
+    faraday_middleware (0.10.0)
```

Comment:
it doesn't appear as though you're using faraday_middleware, i think this is probably going to be used by others, but unclear to me why it should be in this PR if not being used.